service/loggingService/service.go

```diff
@@ -0,0 +1,28 @@
+package loggingservice
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+)
+
+type ContextKey string
+
+const ContextKeyRequestID ContextKey = "requestID"
+
+type LoggingService struct {
```

Comment:
I don't see the reason why we need this as a service and not as a helper func in controller.

Reply:
In two controllers, as well as in the handler. I think it's better like this